README.md

```diff
@@ -1,39 +1,38 @@
 # rocBLAS-Examples
-Examples for using rocBLAS (Radeon Open Compute Basic Linear Algebra Subprograms) which is a GPU exploiting implementation of BLAS. 
+Examples for using rocBLAS (Radeon Open Compute Basic Linear Algebra Subprograms) which is a GPU exploiting implementation of BLAS.
 
 # rocBLAS
-rocBLAS is AMD's library for [BLAS](http://www.netlib.org/blas/) on [ROCm](https://rocm.github.io/install.html). 
-It is implemented in the [HIP](https://github.com/ROCm-Developer-Tools/HIP) 
+rocBLAS is AMD's library for [BLAS](http://www.netlib.org/blas/) on [ROCm<sup>TM</sup>](https://rocm.github.io/install.html).
+It is implemented in the [HIP](https://github.com/ROCm-Developer-Tools/HIP)
 programming language and optimized for AMD's GPUs.
 
 |Acronym      | Expansion                                                   |
 |-------------|-------------------------------------------------------------|
 |**BLAS**     | **B**asic **L**inear **A**lgebra **S**ubprograms            |
-|**ROCm**     | **R**adeon **O**pen **C**ompute platfor**m**                |
 |**HIP**      | **H**eterogeneous-Compute **I**nterface for **P**ortability |
```

Comment:
I think it is `**H**eterogeneous-Computing` referred here:https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html

Reply:
The repo https://github.com/ROCm-Developer-Tools/HIP states it as used.  This doesn't matter for now as don't think it is contentious.